staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -155,6 +157,15 @@ type Config struct {
 	// If specified, long running requests such as watch will be allocated a random timeout between this value, and
 	// twice this value.  Note that it is up to the request handlers to ignore or honor this timeout. In seconds.
 	MinRequestTimeout int
+
+	// If specified, poststarthook healthz checks will return true UNTIL this configured delay elapses.
+	// By default, this value starts at 0, which preserves existing apiserver poststarthook health checking behavior.
+	// You may want to override this because currently, the amount of time allotted for these poststarthooks to complete
+	// is obscure. Specifically, if the poststarthook must complete by the following timeout:
+	// the static pod's configured InitialDelaySeconds + (FailureThreshold * PeriodSeconds)
+	// If none of these values are overridden, this means that if the kube-apiserver boot-sequence does not complete
+	// in 45 seconds, the kubelet will restart the apiserver.
```

Comment:
those 45 seconds depend on the context. Also not every kube-apiserver runs in a static pod.

Reply:
make this shorter and more crisp. Background is nice, but I would remove the whole paragraph about the static pod environment. It's not generic.

Reply:
Ok.